python/test/t_Viewer.py

```diff
@@ -89,14 +89,6 @@
 # view.save('curve7.png')
 view.show(block=False)
 
-# Pie
-graph = ot.SensitivityAnalysis.DrawImportanceFactors(
-    [.4, .3, .2, .1], ['a0', 'a1', 'a2', 'a3'], 'Zou')
-# graph.draw('curve8.png')
-view = View(graph)
-# view.save('curve8.png')
-view.show()
-
```

Comment:
The Pie type is not tested anymore if you remove this.
Can't you use SobolIndicesAlgorithmImplementation::DrawImportanceFactors instead here ?